config.toml

```
@@ -0,0 +1,3 @@
+baseURL = ""
```

Since this site will be published on GitHub Pages, for now, let's use the convention of `unicef.github.io` + `repo_name`as what the base URL for this theme will be. If @enonied4g gets a custom domain or subdomain for the site, we can revisit this part later.

```suggestion
baseURL = "https://unicef.github.io/drone-dpgtoolkit"
```